gender_novels/corpus_gen.py

```diff
@@ -451,6 +501,10 @@ def get_author_gender(authors):
     'female'
     >>> get_author_gender(['Shakespeare, William', "Duan, Mingfei"])
```

Comment:
is 'both' correct in this case, since you're both male?

Reply:
or is gender_guesser not working here! ah!  okay -- feel free to interweave comments with doctest results.  That's what makes Doctests so great. (added a comment above)